Synergy/Validators/PermissionsValidator.cs

```diff
@@ -0,0 +1,19 @@
+using FluentValidation;
+using Synergy.Models;
+
+namespace Synergy.Validators;
+
+public class PermissionsValidator : AbstractValidator<Permissions>
+{
+    public PermissionsValidator()
+    {
+        RuleFor(input => input.PermissionName).NotEmpty().NotNull().Must(text => !text.Contains('$') && !text.Contains('.'))
```

Comment:
This checking to make sure it doesn't contain $ and "." appears to be re-used in other validators, perhaps you chould add a .NotContainCharacter() extension (if there isn't one) to make it a lot easier to read. Also it is the atomic way to new line any .Something() methods for reading so this would be

```
RuleFor(input => input.PermissionName)
.NotEmpty()
.NotNull()
.Must(.... etc
```